src/file_system/error.rs

```diff
@@ -44,10 +44,10 @@ pub(crate) fn label_has_slash(item: &str) -> Error {
 pub enum Error {
     /// A `LabelError` specific error for parsing a
     /// [`Path`](super::path::Label).
-    #[error("Label error: {0}")]
+    #[error("malformed label: {0}")]
     Label(#[from] LabelError),
     /// A `PathError` specific error for parsing a [`Path`](super::path::Path).
-    #[error("Path error: {0}")]
+    #[error("malformed path: {0}")]
     Path(#[from] PathError),
```

Comment:
The `PathError` is already `#[from]`, which implies that it's returned as `source`, so it shouldn't be printed it in the error message.

Same for `Label`.